backend/src/govsg/models/govsg-template.ts

```diff
@@ -0,0 +1,47 @@
+import { Campaign } from '@core/models'
+import {
+  BelongsToMany,
+  Column,
+  DataType,
+  Model,
+  Table,
+} from 'sequelize-typescript'
+import { CampaignGovsgTemplate } from './campaign-govsg-template'
+
+@Table({ tableName: 'govsg_templates', underscored: true, timestamps: true })
+export class GovsgTemplate extends Model<GovsgTemplate> {
+  @Column({
+    type: DataType.BIGINT,
+    autoIncrement: true,
+    primaryKey: true,
+  })
+  id: number
+
+  @BelongsToMany(() => Campaign, {
+    through: { model: () => CampaignGovsgTemplate },
+  })
+  campaign!: Campaign
+
+  @Column({
+    type: DataType.TEXT,
+  })
+  body?: string
```

Comment:
I think the convention is to use `body: string | null` rather `body?: string`, which might screw up TS. (can this field be null though?)

Reply:
This field won't be null actl, lemme just remove